src/I18n/Number.php

```diff
@@ -263,6 +284,31 @@ public static function defaultCurrency($currency = null)
         return self::$_defaultCurrency;
     }
 
+    /**
+     * Getter/setter for default currency format
+     *
+     * @param null $currencyFormat Default currency format to be used by currency()
```

Comment:
wrong types (missing ones)

Reply:
Stickler broke the type, because I had the variable name wrong.